helm/oauth2-proxy/values.yaml

```diff
@@ -463,6 +463,9 @@ redis:
   #
```

Comment:
I think we should take the opportunity and change this section a bit more. Instead of having a single node cluster setup as the recommendation. We should reference the example files directory here. What do you think?

Reply:
Anything that improves the readability or usefulness of the value file is welcome.